autokeras/hypermodel/preprocessor.py

```diff
@@ -416,9 +415,10 @@ class ImageAugmentation(base.Preprocessor):
     """Collection of various image augmentation methods.
 
     # Arguments
+        percentage: Float. The percentage of data to augment.
         rotation_range: Int. The value can only be 0, 90, or 180.
             Degree range for random rotations. Default to 180.
-        random_crop: Boolean. Whether to crop the image randomly. Default to True.
+        crop: Boolean. Whether to crop the image randomly. Default to True.
```

Comment:
Why change the name? `crop` could be interpreted in different ways: for instance, in the preprocessing layers API, we have `CenterCrop` and `RandomCrop`.